frontend/src/pages/projects/screens/detail/connections/ConnectedResources.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { LabelGroup, Spinner } from '@patternfly/react-core';
+import {
+  useRelatedNotebooks,
+  ConnectedNotebookContext,
+} from '~/pages/projects/notebook/useRelatedNotebooks';
+import { Connection } from '~/concepts/connectionTypes/types';
+import { ProjectObjectType } from '~/concepts/design/utils';
+import ResourceLabel from '~/pages/projects/screens/detail/connections/ResourceLabel';
+import { getDisplayNameFromK8sResource } from '~/concepts/k8s/utils';
+
+type Props = {
+  connection: Connection;
+};
+
+const ConnectedResources: React.FC<Props> = ({ connection }) => {
+  const { notebooks: connectedNotebooks, loaded: notebooksLoaded } = useRelatedNotebooks(
+    ConnectedNotebookContext.EXISTING_DATA_CONNECTION,
+    connection.metadata.name,
+  );
+
+  if (!connectedNotebooks.length) {
+    return '-';
+  }
+
+  if (!notebooksLoaded) {
+    return <Spinner size="sm" />;
+  }
```

Comment:
I don't know if the `<Spinner />` will ever return? If it's not loaded, the connectedNotbooks will always be [], so it would always return `'-'` while loading? 

So I think these 2 need to be switched around?